tools/project_manager/editproject.php

```diff
@@ -1031,7 +1031,7 @@ function preview()
         echo "<br>$a<br>$b";
         echo "</th></tr>\n";
         echo "<tr><td colspan='2'>";
-        echo $comments;
+        echo html_safe($comments);
```

Comment:
Alas, HTML is valid in project comments, so this and other outputs of them shouldn't be escaped.

Reply:
updated